hydra-node/src/Hydra/API/Server.hs

```diff
@@ -98,21 +99,18 @@ withAPIServer config env party persistence tracer chain pparams serverOutputFilt
     commitInfoP <- mkProjection CannotCommit projectCommitInfo
     headIdP <- mkProjection Nothing projectInitializingHeadId
     pendingDepositsP <- mkProjection [] projectPendingDeposits
-    loadedHistory <-
-      runConduitRes $
-        source
-          -- .| mapC output
-          .| iterM (lift . atomically . update headStatusP . output)
-          .| iterM (lift . atomically . update snapshotUtxoP . output)
-          .| iterM (lift . atomically . update commitInfoP . output)
-          .| iterM (lift . atomically . update headIdP . output)
-          .| iterM (lift . atomically . update pendingDepositsP . output)
-          -- FIXME: don't load whole history into memory
-          .| sinkList
-
-    history <- newTVarIO loadedHistory
-    -- NOTE: we need to reverse the list because we store history in a reversed
-    -- list in memory but in order on disk
+    let history =
+          sourceEvents
```

Comment:
We should still run this conduit here (to prime the projections) and then pass the conduit to websocket server.